src/meshapi/serializers/map.py

```diff
@@ -149,6 +153,17 @@ class Meta:
     installDate = JavascriptDateField(source="install_date")
 
     def convert_status_to_spreadsheet_status(self, link: Link) -> str:
+        if link.map_display_override:
+            map_override_to_spreadsheet_status: Dict[str, str] = {
```

Comment:
📓 Note: What happens if we update the MapDisplayOverride and forget to update this? Is there a test that evokes that?